ansible/roles/maintenance/tasks/main.yml

```diff
@@ -0,0 +1,108 @@
+- name: Checkout Caracalla
+  git:
+    repo: "{{caracalla_git_repo}}"
+    dest: "{{caracalla_checkout}}"
+    force: yes
+    version: "{{caracalla_branch}}"
+
+# We don't use {{caracalla_checkout}}, because we want to use master
+# branch as base branch. {{caracalla_checkout}} can use different branch
+# e.g. developer adds new functionality in feature branch like you
+# right now [jiri] :-P
+- name: Delete old caracalla git directory (PR)
+  shell: 'rm -rf {{git_dir}}'
+
+- name: Create new directory for Caracalla clone
+  shell: 'mkdir {{git_dir}}'
+
+- name: Checkout Caracalla branch (PR)
+  shell: 'git clone {{git_repo}}'
+  args:
+    chdir: "{{git_dir}}"
+
+- set_fact:
+    git_dir: "{{git_dir}}/caracalla"
+
+- name: Get current date
+  shell: 'date +%Y-%m-%d_%H-%M'
```

Comment:
Instead, you can get this from facts ansible already gathers: `{{ ansible_date_time.date }}_{{ ansible_date_time.hour }}-{{ ansible_date_time.minute }}`.

Reply:
Good point.